### PR TITLE
Collective lift and shift updates

### PR DESCRIPTION
**Pull request by jonrandahl**
This PR includes the following updates and resolves the relevent linked issues:

- Added a cli recommendation for running the `sr-manager` locally;
  resolved a markdown linting issue with using HTML in markdown; as well as
  fixed a simple typo in the `localhost:3001` port example
- Refactors the elapsed time calculated for API requests to be resolved as
  microseconds rather than milliseconds. This is to improve the reporting of the
  elapsed time in the system tooling logs. [#104](https://github.com/epimorphics/hmlr-linked-data/issues/104)/[#111](https://github.com/epimorphics/hmlr-linked-data/issues/111)
- Minor text changes to the `Gemfile` to include instructions for running
  Epimorphics specific gems locally during the development of those gems.
- Updated the production `data_services_api` gem version to be at least
  the current version`~>1.3.3` (this is to cover out of sync release versions)
- Updated the production `lr_common_styles` gem version to be at least the
  current version `~>1.9.1` (this is to cover out of sync release versions)
- Refactored better guards in `entrypoint.sh` to ensure the required env
  vars are set accordingly or deployment will fail noisily.
- Refactored the error messages displayed to be semantically formatted as
  well as incorporated ALL lines for the returned message to be held in the
  variable instead of being possibly displayed outside of the intended context.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,31 @@
...
+- (Jon) Refactors the elapsed time calculated for API requests to be resolved
+  as microseconds rather than milliseconds. This is to improve the reporting of
+  the elapsed time in the system tooling logs.
+- (Jon) Minor text changes to the `Gemfile` to include instructions for running
+  Epimorphics specific gems locally during the development of those gems.
+- (Jon) Updated the production `json_rails_logger` gem version to be at least the
+  current version `~>1.3.5` (this is to cover out of sync release versions)
+- (Jon) Updated the production `lr_common_styles` gem version to be at least the
+  current version `~>1.9.1` (this is to cover out of sync release versions)
+- (Jon) Renamed the global env variable `RAILS_RELATIVE_URL_ROOT` to
```
Comment (der) [inline CHANGELOG.md]: If that definitely works and you find it more intuitive then that's fine.

Just wanted to note that RAILS_RELATIVE_URL_ROOT is a conventional name that you see in e.g. github issues and some of the tool chain, notably webpacker is aware if it. Given rails "convention over configuration" mantra this leaves me a little nervous :)

**Review by der — APPROVED** [review]
Looks fine to me other than unfounded fear, uncertainty and doubt over the renaming of RAILS_RELATIVE_URL_ROOT.

File: CHANGELOG.md (hunk 1 above)
Comment (jonrandahl) [reply CHANGELOG.md]: @der I've resolved the codebase to use the requested `APPLICATION_ROOT` variable name for the build process and the `RAILS_RELATIVE_URL_ROOT` for the app via the `entrypoint.sh` code. 🙏

**Review by bogdanadrianmarc — APPROVED** [review]
Changes look good from a frontend perspective 👍

**Review by der — APPROVED** [review]
:+1:

**Review by andrew-pickin-epi — CHANGES_REQUESTED** [review]
Please fix as pre LR Landing then I will review

**Review by der — APPROVED** [review]
As per others. I don't understand using `SCRIPT_NAME` and `APPLICATION_ROOT` instead of just `RAILS_RELATIVE_URL_ROOT` but once the pattern is resolved on the others assume it'll be applied here so approving.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
+  `APPLICATION_ROOT` to be more clear on it's use in the `entrypoint.sh` code.
```
Comment (andrew-pickin-epi) [inline CHANGELOG.md]: This change log would have us still using APPLICATION_ROOT, which I don't think is correct.

File: Makefile [hunk 3]
```diff
@@ -26,7 +26,7 @@ IMAGE?=${NAME}/${STAGE}
 REPO?=${ECR}/${IMAGE}
 
 GITHUB_TOKEN=.github-token
-BUNDLE_CFG=.bundle/config
+BUNDLE_CFG=${HOME}/.bundle/config
```
Comment (andrew-pickin-epi) [inline Makefile]: This shouldn't be ${HOME}.

**Review by andrew-pickin-epi — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (andrew-pickin-epi) [reply CHANGELOG.md]: Summary of this PR still refers to APPLICATION_ROOT also.

Comment (jonrandahl) [reply CHANGELOG.md]: Removed that line from changelog, thank you!

File: Makefile (hunk 3 above)
Comment (jonrandahl) [reply Makefile]: Removed `${HOME}/` from path!

**Review by andrew-pickin-epi — APPROVED** [review]

**Pull request merged**